src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -0,0 +1,202 @@
+from azext_load.data_plane.utils import validators, completers
+from azure.cli.core.commands.parameters import (
+    get_resource_name_completion_list,
+    quotes,
+    resource_group_name_type,
+)
+from knack.arguments import CLIArgumentType
+
+quote_text = "Use {} to clear existing {{}}.".format(quotes)
+
+# Common arguments
+resource_group = resource_group_name_type
+
+load_test_resource = CLIArgumentType(
+    options_list=["--load-test-resource", "--name", "-n"],
```

Comment:
I don't prefer using `--name,-n` args for the load test resource name since CLI convention is usually to use this param for the name of the entity the command interacts with.
E.x. `load test create -n {some_name}`, here the usual convention is that `{some_name}` will be used as the name of test. Thoughts? 
Okay to discuss/confer with PMs as well

Reply:
Sure, we can discuss. The idea was that `load-test-resource` is a big argument used in every CLI command, so it would be easier to handle if given a short alternate